utils

```diff
@@ -23,7 +23,7 @@ get_project_name() {
    echo $(get_app_name | tr -cd '[[:alnum:]]' | tr /A-Z/ /a-z/)
 }
 
-if [[ -v VIRTUAL_ENV ]]; then
+if [ ! -z ${VIRTUAL_ENV:-} ]; then
```

Comment:
Does this need to be `"${quoted}"`?

Reply:
I don't think so particularly, but happy to be told otherwise. Some bash conventions probably wouldn't hurt us.